src/west/configuration.py

```diff
@@ -70,6 +127,278 @@ class ConfigFile(Enum):
     GLOBAL = 3
     LOCAL = 4
 
+class Configuration:
```

Comment:
`class Configurations`, plural?

Before reading the pydoc carefully I wrongly guessed that there would be 3 separate instances: local, global and system.